intellij/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/EditorWindowListener.java

```diff
@@ -24,14 +23,16 @@
      * events of the editor. By using Disposable for this structure, we can
      * properly cleanup all listeners once we dispose ourselves.
      */
-    private Map<Editor, Disposable> editorDisposableTreeMap = new HashMap<>();
+    private final Map<Editor, Disposable> editorDisposableTreeMap;
```

Comment:
Do these really need to be final?

Reply:
They are never changed nor should they, so this is mostly a hint to the compiler for optimization purposes.

Reply:
I would prefer "final" to be removed if it is not absolutely necessary to compile. It burdens people's mind as to why it can't just be a normal local variable.